python/ray/autoscaler/kubernetes/operator_configs/example_cluster_resource.yaml

```diff
@@ -0,0 +1,109 @@
+apiVersion: cluster.ray.io/v1
+kind: RayCluster
+metadata:
+  name: test-cluster
+spec:
+  targetUtilizationFraction: 0.8
+  idleTimeoutMinutes: 5
+  headPodType: head-node
+  workerDefaultPodType: worker-nodes
+  podTypes:
+  - name: head-node
+    podConfig:
+      apiVersion: v1
+      kind: Pod
+      metadata:
+        generateName: test-cluster-ray-head-
+        labels:
+          component: ray-head
+      spec:
+        restartPolicy: Never
+
+        # This volume allocates shared memory for Ray to use for its plasma
+        # object store. If you do not provide this, Ray will fall back to
+        # /tmp which cause slowdowns if is not a shared memory volume.
+        volumes:
+        - name: dshm
+          emptyDir:
+            medium: Memory
+        containers:
+        - name: ray-node
+          imagePullPolicy: Always
+          image: rayproject/ray:nightly
+          # Do not change this command - it keeps the pod alive until it is
+          # explicitly killed.
+          command: ["/bin/bash", "-c", "--"]
+          args: ['trap : TERM INT; sleep infinity & wait;']
+          ports:
+          - containerPort: 6379 # Redis port.
+          - containerPort: 6380 # Redis port.
+          - containerPort: 6381 # Redis port.
```

Comment:
do we still need 3 redis ports?

Reply:
I think this was just to show how you would do it -- redis uses only a single port by default AFAIK

Reply:
Keeping things simple -- going to remove the two extra ports.